src/consumerIncentives/saga.ts

```diff
@@ -6,13 +6,8 @@ import { showError, showMessage } from 'src/alert/actions'
 import { RewardsEvents } from 'src/analytics/Events'
```

Comment:
you can also pull on the `isSuperchargePendingRewardsV2` thread, there's a claim function that you can remove :)